views/summaryFormExample.html

```diff
@@ -22,35 +22,35 @@ <h1 class="heading-xlarge">
                     <caption class="heading-small">Example form summary</caption>
                     <tbody>
                         <tr>
-                           <td>Full name</td>
+                           <th>Full name</th>
```

Comment:
This was failing accessibility checks before - talked with @KylerD and we agreed that it was appropriate to update these to th's. Styles remain identical and tests now pass